### PR TITLE
Kaivos patch 2

### PR DESCRIPTION
**Pull request by kkmattil**
Update to kaivos manual.

### CODE REVIEW

File: docs/data/kaivos/kaivos-account.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
 # 1. Getting a database account
```
Comment (galfthan) [inline docs/data/kaivos/kaivos-account.md]: All numbers in headings should be removed

**Comment by galfthan** [comment]
Also, this should be added to the menu.

File: docs/data/kaivos/kaivos-account.md (hunk 1 above)
Comment (kkmattil) [reply docs/data/kaivos/kaivos-account.md]: Removed

**Comment by attesillanpaa** [comment]
I accidentally made the changes to an older branch. Copied other changes from this, next PR the one with all changes.

**Pull request closed without merging**